doc/source/contributing/doc_dev_summary.md

```diff
@@ -0,0 +1,51 @@
+(doc_dev_summary)=
+# Building the documentation
+
+The preferred workflow for contributing to ArviZ is to fork
+the [GitHub repository](https://github.com/arviz-devs/arviz/),
+clone it to your local machine, and develop on a feature branch. For a detailed description of the recommended development process, see the step-by-step guide {ref}`here <build_steps>`.
+
+If you are using Docker, see {ref}`building_doc_with_docker`.
+
+## Pull request checks
+
+Every PR has a list of checks that check if your changes follow the rules being followed by ArviZ docs.
+You can check why a specific test is failing by clicking the `Details` next to it. It will take you to errors and warning page. This page shows the details of errors, for example in case of docstrings:
+
+`arviz/plots/pairplot.py:127:0: C0301: Line too long (142/100) (line-too-long)`.
+
+It means line 127 of the file `pairplot.py` is too long.
+
+### Docstring
+
+Docstrings follow the pylint rules. Don't forget to check the following points before opening your PR:
+
+* The maximum number of characters allowed in a line is 100. If a line consists of more than 100 characters, tests will fail.
+* If there is any trailing white space, tests will fail. If you are using visual studio code, you can enable the [Trailing Spaces](https://marketplace.visualstudio.com/items?itemName=shardulm94.trailing-spaces) extension to visualize the trailing spaces.
+
+(preview_change)=
+## Previewing doc changes
+
+There is an easy way to check the preview of docs by using GitHub. Open the PR and GitHub actions will automatically build the documentation.
```

Comment:
very nitpicky comment. it is not github actions but readthedocs what automatically builds the docs. github actions is one of the ways to run ci checks on github, but arviz uses Azure and readthedocs